.editorconfig

```diff
@@ -2,11 +2,8 @@ root = true
 
 [*]
 charset = utf-8
-indent_style = tab
+indent_style = space
```

Comment:
> @woubuc:
> Code should always be indented with tabs, not spaces, and the `indent_size` should not be set. This allows everyone to use the indentation level they prefer.
> 
> The only exception should be .json files, specifically package.json since it gets rewritten by npm/yarn using 2 spaces for indentation.

> @MichaelDeBoey:
> That would lead to inconsistent formatting if people can choose indentation.
> 
> I chose to use spaces, since that's more commonly used nowadays.

> @woubuc:
> The formatting would be tab-based for everyone. People can configure their IDE to display a `tab` character as any width, but in the file that indentation will always be represented by a `tab` character. This makes the code more accessible to a wider range of developers with different requirements.

Nowadays it's more commonly used to use `spaces` (see `@typescript-eslint/recommended` plugin), so I just changed the `.editorconfig` to use these rules.

Reply:
I agree that spaces are more common, but it's not accessible for everyone. Using tab is just as easy for most people, and it makes the code much more accessible. See [this reddit thread](https://www.reddit.com/r/javascript/comments/c8drjo/nobody_talks_about_the_real_reason_to_use_tabs/) (or other articles written about it, like [this one](https://dev.to/alexandersandberg/why-we-should-default-to-tabs-instead-of-spaces-for-an-accessible-first-environment-101f)) for some reasons why I will always advocate for tab-based indentation.